jwst/resample/resample_step.py

```diff
@@ -55,6 +57,7 @@ def process(self, input):
             kwargs = self.get_drizpars(ref_filename, input_models)
         else:
             # Deal with NIRSpec which currently has no default drizpars reffile
+            self.log.info("No NIRSpec DIRZPARS reffile")
```

Comment:
Doubt this is needed, because NIRSpec imaging mode data never get resampled, but can't hurt to include it.

Reply:
You'll notice that `ResampleSpecStep` now subclasses from `ResampleStep`, so this is actually needed for the NIRSpec spectral modes.  I did this to reduce the amount of repeated code between the two classes.

The end, zen state, will be for there to be a single ResampleStep that can handle spectra or images, but I would like to get my test coverage up before I do that, as that's a major refactor.